cpp/src/gandiva/regex_functions_holder_test.cc

```diff
@@ -330,16 +289,14 @@ class TestReplaceHolder : public ::testing::Test {
 };
 
 TEST_F(TestReplaceHolder, TestMultipleReplace) {
-  std::shared_ptr<ReplaceHolder> replace_holder;
-
-  auto status = ReplaceHolder::Make("ana", &replace_holder);
-  EXPECT_EQ(status.ok(), true) << status.message();
+  auto replace_holder = ReplaceHolder::Make("ana");
+  EXPECT_EQ(replace_holder.ok(), true) << replace_holder.status().message();
```

Comment:
Can we use `EXPECT_OK_AND_ASSIGN()` here?

Reply:
Sure. I changed some of such usages in these several test files now I change more such usages I find.